frontend/public/src/components/ProgramEnrollmentDrawer.js

```diff
@@ -49,9 +49,9 @@ export class ProgramEnrollmentDrawer extends React.Component<ProgramEnrollmentDr
       <>
         <div className={backgroundClass}>
           <div className={drawerClass} id="program_enrollment_drawer" tabIndex="-1" role="dialog" aria-modal="true" aria-label="program courses" aria-describedby="program_enrolled_items">
-            <div className="row chrome" id="program_enrollment_title">
-              <h3>{enrollment.program.title}</h3>
-              <button type="button" className="close float-right" aria-label="Close" onClick={closeDrawer}>
+            <div className="row chrome d-flex flex-row" id="program_enrollment_title">
+              <h3 className="flex-grow-1">{enrollment.program.title}</h3>
+              <button type="button" className="close" aria-label="Close" onClick={closeDrawer}>
                 <span aria-hidden="true">
```

Comment:
Should this have aria-hidden="true" since the user should be able to select it when the drawer is open?

Reply:
Hmm. I changed it to throw the `aria-hidden` on the drawer's wrapper div and tied its value to the state of the drawer; that seemed like the better place for it. Still seems to work with VoiceOver.

Reply:
Yeah I wasn't sure, since the screen reader still worked and you could 'tab' to it.